robusta_krr/core/runner.py

```diff
@@ -253,17 +259,18 @@ async def _collect_result(self) -> Result:
 
             scans = await asyncio.gather(*scans_tasks)
 
+        successful_scans = [scan for scan in scans if scan is not None]
+
         if len(scans) == 0:
             logger.warning("Current filters resulted in no objects available to scan.")
             logger.warning("Try to change the filters or check if there is anything available.")
             if settings.namespaces == "*":
                 logger.warning(
                     "Note that you are using the '*' namespace filter, which by default excludes kube-system."
                 )
-            return Result(
-                scans=[],
-                strategy=StrategyData(name=str(self._strategy).lower(), settings=self._strategy.settings.dict()),
-            )
+            raise CriticalRunnerException("No objects available to scan.")
```

Comment:
how do these exceptions look in runner?

Reply:
They write a critical log in the end and exit with error